src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -193,6 +201,26 @@ private List<string> ResolveByExternalReferencePackages(List<string> uidList, Co
             return list;
         }
 
+        private XRefSpec FindByRequest(string uid)
+        {   
+            
+            XRefSpec xf = null;
+            string requestUrl = XRefServerUrls[0];
+            string url = requestUrl + "/" + uid + "/";
```

Comment:
should allow following request:
[http://consoto.com/uids/some-uid?tags=dotnet-framwork](http://consoto.com/uids/some-uid?tags=dotnet-framwork)

---
In reply to: [125900125](https://github.com/dotnet/docfx/pull/1837#discussion_r125900125) [](ancestors = 125900125)